### ISSUE TITLE
I don't know how to implement transaction with Waterline

### ISSUE THREAD

**Issue by jyoungrok** [balderdashy/sails#2858]
please refer to my posts i wrote down before
(https://github.com/balderdashy/waterline/issues/960)
(https://github.com/balderdashy/waterline/issues/614)

If sails framework doesn't have solution for that, i might make new connection to mysql like below

var client=mysql.createConnection({});

However, what i concerned with is that if i make new connection ,there will be 2 connection to mysql (original connection in Sails and new connection i will make )
so if i do that, will there be any problem?

please comment it 
thank you

**Comment by sailsbot** [comment]
Thanks for posting, @jyoungrok.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
